client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -183,6 +184,20 @@ UndocumentedSite.prototype.setOption = function( query, callback ) {
 	);
 }
 
+UndocumentedSite.prototype.postCounts = function( options, callback ) {
+	var type,
+		query = merge( {
+			type: 'post',
+			apiNamespace: 'wp', // This currently has no bearing on the route, wpcom-xhr-request needs to be updated to remove the namespace whitelist
+			locale: i18n.getLocaleSlug()
```

Comment:
Since this endpoint doesn't need locale, thinking I might just remove this logic here.